docs/blogs/reshare.html

```diff
@@ -0,0 +1,206 @@
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol"> </span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="edge-logo-symbol">*</span>
+        </div>
+        <div class="logo-line">
+            <span class="vertical-logo-symbol">|</span>
+            <span class="logo-text"> commonware </span>
+            <span class="vertical-logo-symbol"> </span>
+        </div>
+        <div class="logo-line">
+            <span class="edge-logo-symbol">*</span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol"> </span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol"> </span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">*</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="edge-logo-symbol">+</span>
+        </div>
+    </div>
+    <div class="content">
+        <h1>Reconfiguration + Reshare with threshold <code>consensus::simplex</code></h1>
+        <div class="meta">
+            <div class="author">By <a href="https://x.com/vex_0x">Ben Clabby</a></div>
+            <div class="date">October 15, 2025</div>
+        </div>
+        <p>
+            An ongoing challenge with maintaining distributed validator sets is reconfiguring the validator committee
+            (adding and removing validators), and in turn, refreshing secret key material amongst active participants.
+            Networks need to expand and contract the committee without touching the brakes; Validator committees are
+            rarely static in practice, e.g. to allow for staking/unstaking on Proof of Stake networks, or rotating the
+            authorities on Proof of Authority networks.
+        </p>
+        <p>
+            In many modern blockchains, reconfiguring the committee still means injecting out-of-band key material,
+            with each validator responsible for the secure generation and safekeeping of their own signing keys. Some
+            validators may have done their due diligence, but some may have not, hindering the security properties of
+            the network at large. What if there was a way to enforce validators to refresh their key material on a
```

Comment:
nit: this is typically called "proactive refresh" in case you want to weave this in anywhere